docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,89 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+- [ ] Remove all drives
+- [ ] Switch drive selection
+- [ ] Insert a single drive, and expect auto-selection
+- [ ] Insert one system drive, and don't expect auto-selection
+
+Image Support
+-------------
+
+- [ ] Flash an uncompressed image
+- [ ] Flash a Bzip2 image
+- [ ] Flash a XZ image
+- [ ] Flash a ZIP image
+- [ ] Flash a GZ image
+- [ ] Flash a DMG image
+- [ ] Flash an image whose size is not a multiple of 512 bytes
```

Comment:
And based on the bugs found during the testing of the recent release, this should both test an uncompressed image which isn't a multiple of 512 bytes, and an archive which contains an image which isn't a multiple of 512 bytes.

Reply:
Added!